lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
+            repo = oelite.git.GitRepository(self.repo)
+            if self.has_rev(repo):
```

Comment:
We decided to duplicate the logic from fetch, since we where not sure if you could hit this without running fetch as well. Personally i think the check belongs higher in the hierachy. We can just make it return True if you prefer that.